docs/operations.md

```diff
@@ -28,6 +28,138 @@ Clients may inspect this specification at run-time so to obey the various limits
+  - `ADD_TAGS`: tags were added to a job
+  - `DELETE_TAGS`: tags were removed from a job
+  - `UPDATE_STATUS`: job status has been updated
+  - `UPDATE_DEFINITION`: job definition has been updated
+- `<JOB>` is the JSON representation of the job associated with this event
+- `<EVENT_ID>`: a (unique) numeric identifier for the event, starting at 1 and incremented by 1 for each subsequent event; can be used to detect message loss on the client side.
+
+**Example:**
+
+```
+data: {"action":"UPDATE_STATUS","job":{"clientId":"Dana","definition":{"title":"Expose Job API"},"history":null,"id":"d305e539-1d41-4c95-b19a-2a7055c469d0","mtime":"2023-08-29T13:00:57.632+02:00","status":{"definitionHash":"f8b3d6ea0150a98ec5f8dc224659211a37cb4737777388f77a3e323a865e7674","state":"PROGRESS"},"stime":"2023-08-22T15:30:02.323+02:00","tags":[],"workflow":{"groups":null,"name":"wfx.workflow.kanban","states":null,"transitions":null}}}
+id: 1
+
+```
+
+#### Filter Parameters
+
+Job events can be filtered using any combination of the following parameters:
+
+- Job IDs
+- Client IDs
+- Workflow Names
+
+This enables more precise control over the events which are dispatched.
+
+#### Examples
+
+`wfxctl` offers a reference client implementation. The following command subscribes to **all** job events:
+
+```bash
+wfxctl job events
+```
+
+This may result in a large number of events, though. For a more targeted approach, filter parameters may be used.
+Assuming the job IDs are known (either because the jobs have been created already or the IDs are received via another
+subscription channel), the following will subscribe to events matching either of the two specified job IDs:
+
+```bash
+wfxctl job events --ids d305e539-1d41-4c95-b19a-2a7055c469d0,e692ad92-45e6-4164-b3fd-8c6aa884011c
+```
+
+See `wfxctl job events --help` for other filter parameters, e.g. workflow names.
+
+#### Considerations and Limitations
+
+1. **Asynchronous Job Status Updates**: Job status updates are sent asynchronously to avoid the risk of a subscriber
+   causing delays. As a result, messages could arrive in a not necessarily linear order, deviating from the client's
+   expectation of a naturally ordered flow of events. While this is generally not a concern, it could become an issue in
+   high-concurrency situations. For example, when multiple clients try to modify the same job or when a single client
+   issues a rapid sequence of status updates.
+2. **Unacknowledged Server-Sent Events (SSE)**: SSE operates on a one-way communication model and does not include an
+   acknowledgment or handshake protocol to confirm message delivery. This design choice aligns with the fundamental
+   principles of SSE but does mean that there's a possibility some events may not reach the intended subscriber (which
+   the client can possibly detect by keeping track of SSE event IDs).
+3. **Load Balancer Stickiness for Horizontal Scaling**: When scaling out horizontally, ensure that the load balancer is
```

Comment:
This is not directly related to load balancing but an orchestration topic: SSE yield the events for *one* wfx instance only. Now, if there are multiple wfx instances, a consolidated "global" event stream can only be assembled by subscribing to *all* wfx instances (and aggregating the events).

If there's a load balancer in between, there has to be some way to "talk" to specific wfx instances in order to subscribe to their event streams. Hence, there should be a way to sidestep the load balancer to subscribe to specific wfx instances for the `/jobs/events` endpoint, for example.